cmd/tools.go

```diff
@@ -0,0 +1,17 @@
+// +build tools
+
+package cmd
+
+import (
```

Comment:
Go best practices advise to comment on `_` imports to inform the reader of their... importance.